include/libsemigroups/words.hpp

```diff
@@ -56,11 +56,32 @@ namespace libsemigroups {
   // Words
   ////////////////////////////////////////////////////////////////////////
 
-  //! \defgroup Words "Words"
+  //! \defgroup words_group Words
   //! This page contains details of the functionality in \c libsemigroups
   //! related to generating words in a given range and in a given order.
+  //!
+  //! This file contains documentation for functionality for:
+  //!
+  //! * generating words and strings in a given range and in a certain order:
+  //!  - \ref Words
+  //!  - \ref Strings
+  //!  - \ref random_word
+  //!
+  //! * counting words:
+  //!
+  //!   - \ref number_of_words
+  //!
+  //! * converting to and from strings and words:
+  //!
+  //!   - \ref to_string
+  //!   - \ref to_word
```

Comment:
This, for some reason, doesn't create a link

Reply:
Fixed!